### PR TITLE
Change in index.html and style.css file to add work section and about…

### PR DESCRIPTION
**Pull request by ibugithub**
... In this feature branch: 
-We add a work section with 4 cards in index.html
-We add an About me section in index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,6 +77,252 @@ <h3>LET'S CONNECT</h3>
...
+                <ul>
+                    <li><a class="tdnone color396d" href="">HTML</a></li>
+                    <li><a class="tdnone color396d" href="">CSS</a></li>
+                    <li><a class="tdnone color396d" href="">JavaScript</a></li>
+                </ul>
+            </div>
+
+            <div class="see_project">
+                <a class="tdnone color396d" href=""> See project</a>
+            </div>
```
Comment (ShadyShawkat) [inline index.html:151]: - [ ] Kindly ensure that the buttons follow the button styling: Active, Hover, and Pressed (see design)

image

File: index.html [hunk 2]
```diff
@@ -77,6 +77,252 @@ <h3>LET'S CONNECT</h3>
...
+
+        <div class="work1">
+            <div class="work_image">
+                <a href="">
+                    <img src="images/work_image1.png" alt="">
+                </a>
+            </div>
+            <div class="work_title">
+                <h1>Tonic</h1>
+            </div>
```
Comment (ShadyShawkat) [inline index.html:90]: - [ ] Please make sure that there is only one `h1` on the page which is the main title. All other header titles should follow the headers hierarchy of the page (`h2`-`h6`)

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @ibugithub, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — APPROVED** [review]
Hi @ibugithub ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Review by didierganthier — APPROVED** [review]
Hi @ibugithub ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**